app/repositories/recommendation_repository.py

```
@@ -59,3 +66,118 @@ def log_recommendations_batch(self, log_docs: List[Dict[str, Any]]) -> None:
+        """상호작용 데이터 저장 (upsert)"""
+        try:
+            doc = {
+                "recommendation_id": recommendation_id,
+                "user_id": user_id,
+                "paper_id": paper_id,
+                **interaction_data,
+                "updated_at": datetime.utcnow(),
+            }
+
+            # created_at은 최초 생성시에만
+            result = self.interactions_collection.update_one(
```

Flake8 (7.3.0)

[error] local variable 'result' is assigned to but never used

(F841)